pkg/features/embedded/boot-assessment/etc/systemd/system/elemental-boot-assessment.service

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=Elemental system configuration
+After=network-online.target systemd-logind.service
+Wants=network-online.target systemd-logind.service
+ConditionPathExists=/run/elemental/active_mode
+
+StartLimitAction=reboot
+StartLimitIntervalSec=300
+StartLimitBurst=5
```

Comment:
If there are more than 5 failures within a time frame of 5min this will trigger a reboot.